app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/network/MarvelApi.kt

```diff
@@ -0,0 +1,17 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.network
```

Comment:
I think I agree that this should be renamed, but model is a problematic name while it most often refer to data model. I think that is is data layer and it should be renamed to data or repository.
https://github.com/android10/Android-CleanArchitecture
I also think that presenters should be kept together with views (Activities and Fragments) as long as it is not multi-platform project, because it is much simpler to navigate and read, while we often work on them both. 
What do you think?